src/test/mediapipeflow_test.cpp

```diff
@@ -95,6 +95,66 @@ class MediapipeFlowDummyTest : public MediapipeFlowTest {
     }
 };
 
+class MediapipeFlowDummySeparateConfigTest : public MediapipeFlowTest {
+public:
+    void SetUp() {
+        SetUpServer("/ovms/src/test/mediapipe/config_mediapipe_dummy_adapter_full_separate_config.json");
+    }
+};
+
+TEST_P(MediapipeFlowDummySeparateConfigTest, Infer) {
```

Comment:
Why this test needs to be parametrized?

Reply:
Other tests were parametrized to verify support for 1 or more inputs (dummy vs add). I don't see need here?